src/functions/postResult/framework/handler.ts

```diff
@@ -19,7 +20,13 @@ export async function handler(event: APIGatewayProxyEvent, fnCtx: Context): Prom
 
   try {
     const testResult: StandardCarTestCATBSchema = decompressTestResult(body as string);
-    await saveTestResult(testResult);
+    const jwt = event.headers['Authorization'];
+
+    if (examinerCheck(testResult, jwt)) {
+      await saveTestResult(testResult);
+    } else {
+      console.error(`Examiner ID in test and JWT do not match`);
```

Comment:
In this case we don't allow the save but return a success response. Shouldn't we be returning an error code instead